TypeScript.YAML-tmLanguage

```diff
@@ -2443,9 +2443,10 @@ repository:
   type-fn-type-parameters:
     patterns:
     # constructor type
-    - begin: '{{startOfIdentifier}}(new)\b(?=\s*\<)'
+    - begin: '{{startOfIdentifier}}(?:(abstract)\s+)?(new)\b(?=\s*\<)'
```

Comment:
Need to add it at https://github.com/microsoft/TypeScript-TmLanguage/pull/861/files#diff-f920686662480359e6be0b6446e4dde96319b28e5135f8216b43c389023fc157L2454